tests/functional/s3/test_sync_command.py

```diff
@@ -132,25 +132,6 @@ def test_turn_off_glacier_warnings(self):
         self.assertEqual(self.operations_called[0][0].name, 'ListObjectsV2')
         self.assertEqual('', stderr)
 
-    def test_warning_on_invalid_timestamp(self):
```

Comment:
Looking over the commit it got added in: https://github.com/aws/aws-cli/commit/630ae1d9c83744f4df980b94d799d490dc9ead97#, I don't feel comfortable removing this test as there is not really anything to replace it. If there is not a great way to reproduce this at a system level anymore, I'd even be fine with just patching the `get_file_stat()` utility method for this test and adding a todo to find a better way to test it in the future with out patching.